sources/packages/backend/apps/api/src/services/application-offering-change-request/application-offering-change-request.service.ts

```diff
@@ -400,4 +428,56 @@ export class ApplicationOfferingChangeRequestService {
       return applicationOfferingChangeRequest;
     });
   }
+
+  /** Validates student authorization for the provided application offering change request.
+   * @param applicationOfferingChangeRequestId application offering change request id.
+   * @param studentId student id for the authorization.
+   * @returns true if the application offering change request belongs to the student, otherwise false.
+   */
+  async validateStudentForOfferingChangeRequest(
```

Comment:
do we need this service, can't the authorization be achieved by 
![image](https://github.com/bcgov/SIMS/assets/77353155/fccb5474-fe0d-4887-8a82-7e0fda8cc392)

Reply:
removed it.